### PR TITLE
android: Add wop_targets.xml for Halium 7.1 build

### PR DESCRIPTION
**Pull request by Herrie82**
-For Xiaomi Redmi Note 4 (mido) as initial target.

wop_targets.xml: Add vendor/xiaomi, device/xiaomi/mido, kernel/xiaomi/msm8953.
default.xml: Add device/qcom/sepolicy, device/qcom/common,
vendor/qcom/opensource/cryptfs_hw,
external/ant-wireless/antradio-library

Signed-off-by: Herman van Hazendonk <github.com@herrie.org>

### CODE REVIEW

File: default.xml [hunk 1]
```diff
@@ -37,6 +37,9 @@
   <project path="development" name="android_development" remote="los" />
 
   <project path="device/google/contexthub" name="device/google/contexthub" groups="device,marlin,pdk" remote="aosp" />
+  <project path="device/qcom/sepolicy" name="android_device_qcom_sepolicy" groups="qcom" remote="los" />
+  <project path="device/qcom/common" name="android_device_qcom_common" groups="qcom" remote="los" />
+  <project path="vendor/qcom/opensource/cryptfs_hw" name="android_vendor_qcom_opensource_cryptfs_hw" groups="qcom" remote="los" />
```
Comment (Tofee) [inline default.xml:43]: please add these in our local manifest; we should keep Halium's manifest as-is, to keep track of the specific changes we did for our targets.

File: default.xml [hunk 2]
```diff
@@ -51,6 +54,7 @@
   <project path="external/fec" name="platform/external/fec" groups="pdk" remote="aosp" />
   <project path="external/squashfs-tools" name="platform/external/squashfs-tools" groups="pdk" remote="aosp" />
   <project path="external/aac" name="android_external_aac" remote="los" groups="pdk" />
+  <project path="external/ant-wireless/antradio-library" name="android_external_ant-wireless_antradio-library" remote="los" />
```
Comment (Tofee) [inline default.xml:58]: Same here, please add this in our local manifest

**Review by Tofee — CHANGES_REQUESTED** [review]
Apart from the 2 comments, the local manifest looks good

File: default.xml (hunk 1 above)
Comment (Herrie82) [reply default.xml:43]: Well actually it seems to be missing from Halium 7.1 while it should be there. Please see today's Halium log. I already suggested to add them: http://logs.nslu2-linux.org/livelogs/halium.txt just bshah has some reservations.

Comment (Tofee) [reply default.xml:43]: Ah ok; then we should PR it nevertheless, and if the PR is refused we'll move it to our local manifest.

Comment (Tofee) [reply default.xml:43]: In the meantime, let's take your PR as-is.

**Comment by Herrie82** [comment]
Reworked repo and PR-ed upstream.

**Pull request closed without merging**